<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/resources/壁纸.png);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="top_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>116</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1280</width>
       <height>116</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/顶栏壁纸.png);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>43</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>430</width>
          <height>116</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/LOGO2.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;


color: #FFFFFF;
background:transparent;
</string>
        </property>
        <property name="text">
         <string>***</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>116</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="time">
           <property name="styleSheet">
            <string notr="true">font: 40pt &quot;Microsoft YaHei UI&quot;;


color: #FFFFFF;
background:transparent;
</string>
           </property>
           <property name="text">
            <string>15:38</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="date">
           <property name="styleSheet">
            <string notr="true">font: 25pt &quot;Microsoft YaHei UI&quot;;


color: #FFFFFF;
background:transparent;
</string>
           </property>
           <property name="text">
            <string>11月12日</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="type">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;Microsoft YaHei UI&quot;;


color: #FFFFFF;
background:transparent;</string>
        </property>
        <property name="text">
         <string>刚性车</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="state_stack">
        <property name="minimumSize">
         <size>
          <width>430</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QLabel" name="state">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>430</width>
            <height>116</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>80</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>430</width>
            <height>116</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/状态灯_蓝2.png);
font: 30pt &quot;Microsoft YaHei UI&quot;;
color: #FFFFFF;
</string>
          </property>
          <property name="text">
           <string>运转正常</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>430</width>
            <height>116</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>430</width>
            <height>116</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/118.png);
font: 30pt &quot;Microsoft YaHei UI&quot;;
color: #FFFFFF;
</string>
          </property>
          <property name="text">
           <string>实时故障</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QWidget" name="left_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>688</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>688</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>43</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>29</number>
       </property>
       <property name="verticalSpacing">
        <number>56</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_home">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btn_home{
	background-image: url(:/0_4.png);
border:none;
}
QPushButton#btn_home:pressed{
background-image: url(:/按键_返回主界面_按下2.png);
border:none;
}
	</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_weight">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btn_weight{
background-image: url(:/侧键_称重2.png);
border:none;
}
QPushButton#btn_weight:pressed{
background-image: url(:/侧键_称重_按下2.png);
border:none;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_video">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btn_video{
	background-image: url(:/侧键_影像2.png);
border:none;
}
QPushButton#btn_video:pressed{
background-image: url(:/侧键_影像_按下2.png);
border:none;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_engine">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/发动机.png);
border:none</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="c_stackedWidget">
      <property name="minimumSize">
       <size>
        <width>1080</width>
        <height>652</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1080</width>
        <height>652</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/壁纸.png)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
     <widget class="QWidget" name="right_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>688</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>688</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>43</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>29</number>
       </property>
       <property name="verticalSpacing">
        <number>56</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_gearbox">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/gearbox1.PNG);
border:none;
	</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_back">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btn_back{
background-image: url(:/侧键_回退2.png);
border:none;
}
QPushButton#btn_back:pressed{
	
	background-image: url(:/侧键_回退_按下2.png);

border:none;
}
	</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_help">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btn_help{
background-image: url(:/侧键_帮助2.png);
border:none;
}
QPushButton#btn_help:pressed{
	
	background-image: url(:/侧键_帮助_按下2.png);

border:none;
}
	</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_pressure">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/压力.png);
border:none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
